wger/weight/views.py

```diff
@@ -100,6 +103,40 @@ def get_success_url(self):
         '''
         return reverse('weight:overview', kwargs={'username': self.object.user.username})
 
+@login_required
+def get_fitbit_token(request):
+    print("dkbfvkdfbvkdfbvkbdfkvbdfkvbdkfbvkdfbvkdfbvjkdfbvkdfjbvkjdfbvkdfbvkjd")
+    return HttpResponse(
+        "<script src='{src}'></script>".format(
+            src=staticfiles.static('./../static/js/get_fitbit_token.js')
+        ))
+
+@login_required
+def getweight(request, token=None):
+    if token:
+     
+        client_secret = os.getenv('FITBIT_SECRET')
+        client_key = os.getenv('228LLX')
+        token = token
+
+        authorized_client = fitbit.Fitbit(client_key, client_secret,
+                                     access_token=token, refresh_token=token)
+        
+        if authorized_client.get_bodyweight()['weight']:
+            weight_data = authorized_client.get_bodyweight()['weight'][0]
+            date = weight_data['date']
+            weight = weight_data['weight']
+
+            print("*"*100)
+            print(date, weight)
+            print("*"*100)
+            try:
+                weight_object = WeightEntry.objects.create(date=date, weight=weight, user=request.user)
+                weight_object.save()
+            except Exception:
+                pass
```

Comment:
Avoid `pass`ing exceptions. Read more [here](https://stackoverflow.com/a/21553825)
> So the bottom line is: Catch only exceptions you really expect and are prepared to recover from; all others are likely either mistakes you should fix, or something you are not prepared for anyway. Passing specific exceptions is fine if you really don’t need to do something about them.

Reply:
Thanks for pointing this out. Will refactor this.